packages/react/src/components/participant/ParticipantTile.tsx

```diff
@@ -93,14 +93,11 @@ export function ParticipantContextIfNeeded(
  *
  * @example
  * ```tsx
- * {...}
- *   <ParticipantTile>
- *     {...}
- *   </ParticipantTile>
- * {...}
+ * <ParticipantTile>
+ *   ...
```

Comment:
while it's possible to use the tile as a container, I wouldn't use it like this in the example. I think we should point users to use it like 
```
<ParticipantTile source={Track.Source.Camera}`/>
```